### PR TITLE
[46] Product Page using HTML - Mobile Track

### PR DESCRIPTION
**Pull request by Oluwabusiolami**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	  	  
+	  <li><a href="#">Cambridge Global English Workbook Stage 8</a></li>
+	  <h5>$23.30</h5>
+
+ 	  <li><a href="#">Cambridge Global English Teachers Resource Stage 7</a></li>
+	  <h5>$40.50</h5>
+   
+          </li>
+    </main>
+  <footer><a href="#"> Conditions of Use     Privacy Notice    &copy 2019-2020, Justify.com</footer>
```
Comment (ainobs) [inline submissions/oluwabusolamiEluyela/html/index.html:73]: WELL-DONE...
The closing tag of your body is supposed to come after this. Also, please add the image implementation of your task

**Review by ainobs — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 1 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:73]: Which picture am I supposed to add: the image of my website or the html code I used?

Comment (ainobs) [reply submissions/oluwabusolamiEluyela/html/index.html:73]: Image of the website from your browser

Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:73]: ok...thank you

Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:73]: I have reviewed it. Please check my NEW PR.

**Pull request closed without merging**